specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -64,6 +64,325 @@
+            "x-ms-client-name": "Facets",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"$filter",
+            "in": "query",
+            "type": "string",
+            "description": "The OData $filter expression to apply to the search query.",
+            "x-ms-client-name": "Filter",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"highlight",
+            "in": "query",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "The comma-separated list of field names to use for hit highlights. Only searchable fields can be used for hit highlighting.",
+            "x-ms-client-name": "HighlightFields",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"highlightPostTag",
+            "in": "query",
+            "type": "string",
+            "description": "A string tag that is appended to hit highlights. Must be set with HighlightPreTag. Default is &lt;/em&gt;.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"highlightPreTag",
+            "in": "query",
+            "type": "string",
+            "description": "A string tag that is prepended to hit highlights. Must be set with HighlightPostTag. Default is &lt;em&gt;.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"minimumCoverage",
+            "in": "query",
+            "type": "number",
+            "format": "double",
+            "description": "A number between 0 and 100 indicating the percentage of the index that must be covered by am autocomplete query in order for the query to be reported as a success. This parameter can be useful for ensuring search availability even for services with only one replica. The default is 80.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"$orderby",
+            "in": "query",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "The comma-separated list of OData $orderby expressions by which to sort the results. Each expression can be either a field name or a call to the geo.distance() function. Each expression can be followed by asc to indicate ascending, and desc to indicate descending. The default is ascending order. Ties will be broken by the match scores of documents. If no OrderBy is specified, the default sort order is descending by document match score. There can be at most 32 Orderby clauses.",
```

Comment:
`"collectionFormat": "csv"`? Or is that the default?

Reply:
This comment (and also our SDK docs) is out of date. It mentions `geo.distance()`, but it should also mention `search.score()`.

Reply:
Can we link to the OData syntax article here?

Reply:
No.. 

---
In reply to: [208102408](https://github.com/Azure/azure-rest-api-specs/pull/3578#discussion_r208102408) [](ancestors = 208102408)